### PR TITLE
Fixes part of #2646: High-fi Feedback-Item

### PR DESCRIPTION
**Pull request by Mehul-Bisht**
## Explanation

Fixes part of #2646: Changed the dimens files w.r.t Feedback Item

The aim of this PR is just to lay the base for other PRs to come so that it comes easy to review other PRs.

In the end this PR will be solving the issue mentioned in #2554

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by Mehul-Bisht** [comment]
@rt4914 @ARJUPTA This PR changes the dimens w.r.t feedback item, similar to what was done with dimens w.r.t Content Items in #2608. Kindly review this PR.

**Review by FareesHussain — APPROVED** [review]
LGTM.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

@Mehul-Bisht Please do not progress on this PR further. Wait for my PR related to Content-View and based on that this overall issue can be progressed. It will take some time.

File: app/src/main/res/layout-sw600dp-port/feedback_item.xml [hunk 1]
```diff
@@ -12,6 +12,22 @@
       type="org.oppia.android.app.player.state.itemviewmodel.FeedbackViewModel" />
   </data>
 
+  <!-- EXPLORATION SPLIT VIEW -->
```
Comment (chrk2205) [inline app/src/main/res/layout-sw600dp-port/feedback_item.xml:15]: @Mehul-Bisht 
nit changes
maintain either All Capitals or only First Word Capital
Half of the files here have All Capital And the remaining half have Only first world capital

Comment (rt4914) [reply app/src/main/res/layout-sw600dp-port/feedback_item.xml:15]: The files which are in `layout` folder contains caps to make it focusable easily and `dimen` files contains first captial as its just normal comments.

**Pull request merged**